static/js/app.js

```diff
@@ -226,7 +234,7 @@ $(function() {
         },
         fetchMarkers : function() {
             if (!this.isReady) return;
-            this.updateUrl();
```

Comment:
when moving and changing zoom, map fire the idle event, than it's re-fetching the markers, which on success trigger the reloadSidebar that calling the chooseMarker, then when there is no marker selected, it'll call updateUrl.
The udpateUrl does not fire when there is a marker selected and we do map manipulation (without closing the marker info window) do you think it's a problem?